packages/dropdown/src/__stories__/stories.js

```diff
@@ -2,10 +2,20 @@ export default [
   {
     description: "default",
     getProps: () => ({
+      instructions: "Choose one HIG theme to apply to your entire app.",
       label: "HIG Theme",
+      options: ["HIG Light Theme", "HIG Dark Blue Theme", "Matrix Theme"],
+      placeholder: "Select a theme"
+    })
+  },
+  {
```

Comment:
As with the other components, we don't need to create new stories for cases where a prop knob is sufficient.

Reply:
Right, good point where it's a bool like this or other types with limited options.   Updated in 093ba9cf